### ISSUE TITLE
Make Python boilerplate optional in initial setup

### ISSUE THREAD

**Issue by pjbull** [drivendata/cookiecutter-data-science#49]
This makes the project cleaner for other languages, e.g. R, Julia, etc.

**Comment by chrisjkuch** [comment]
Specifically, remove everything in the module folder that is python specific:

```
├── {{ cookiecutter.module_name }}                <- Source code for use in this project.
│   │
│   ├── __init__.py    <- Makes {{ cookiecutter.module_name }} a Python module
│   │
│   ├── data           <- Scripts to download or generate data
│   │   └── make_dataset.py
│   │
│   ├── features       <- Scripts to turn raw data into features for modeling
│   │   └── build_features.py
│   │
│   ├── models         <- Scripts to train models and then use trained models to make
│   │   │                 predictions
│   │   ├── predict_model.py
│   │   └── train_model.py
│   │
│   └── visualization  <- Scripts to create exploratory and results oriented visualizations
│       └── visualize.py
```

should become 

```
├── {{ cookiecutter.module_name }}                <- Source code for use in this project.
│   │
│   ├── data           <- Scripts to download or generate data
│   │
│   ├── features       <- Scripts to turn raw data into features for modeling
│   │
│   ├── models         <- Scripts to train models and then use trained models to make
│   │
│   └── visualization  <- Scripts to create exploratory and results oriented visualizations
```

**IF** the user specifies that they do not want any Python boilerplate.

**Comment by chrisjkuch** [comment]
Closed by #286
